packages/dapp/src/pages/ViewInvoice.jsx

```diff
@@ -316,6 +340,18 @@ export const ViewInvoice = ({
                 <AccountLink address={resolver} chainId={invoiceChainId} />
               </WrapItem>
             </Wrap>
+            <Wrap>
+              <WrapItem>
+                <Text>{'Non-Client Deposits Enabled: '}</Text>
```

Comment:
Similar comment about text here, what should we call this so it is clear what we're disabling.

Reply:
sounds good for all of these